functorch/test/test_vmap.py

```diff
@@ -3807,6 +3807,17 @@ def _assert_all_slices_unique(self, tensor):
         slices_equal.diagonal().zero_()
         self.assertEqual(slices_equal, torch.zeros_like(slices_equal))
 
+    def _assert_correct_size(self, fn, input_tensor, other_args, batch_dim, batch_size, batched_output):
```

Comment:
A note on this: So the actual reason we didn't catch this error with multinomial is because of an issue in the testing (fixed in L4374). But since I had already written this, I added it to the dropout tests. Also happy to split these into two PRs since the multinomial error was just an error in the testing

Reply:
Moved to another PR